api/notd/model.py

```diff
@@ -20,6 +20,18 @@ class RetrievedTokenTransfer:
     blockHash: str
     blockDate: datetime.datetime
 
+@dataclasses.dataclass
+class TokenMetadata:
```

Comment:
would just call this `Token`

Reply:
We already have a class Token

Reply:
cool this is good then. Please update the table name in the sql to `tbl_token_metadatas` then (add the s at the end)